README.md

```diff
@@ -579,12 +501,17 @@ Below are listed all the variables you can customize (you may also want to take
     - openwisp.openwisp2
   vars:
     # openwisp-controler version
-    openwisp2_controller_version: "0.4"
+    openwisp2_controller_version: "0.8.1"
     # optional openwisp2 modules
     openwisp2_network_topology: false
-    openwisp2_network_topology_version: "0.4"
+    openwisp2_network_topology_version: "0.5.1"
     openwisp2_firmware_upgrader: false
     openwisp2_firmware_upgrader_version: "0.1"
+    openwisp2_radius_version: "0.1"
+    # Enable the modules you want to use
+    openwisp2_network_topology: true
+    openwisp2_firmware_upgrader: true
+    openwisp2_radius: true
```

Comment:
what about putting the default values here? Maybe that's what users expect and probably we should (after this PR) update this section to make it explicit.
I'm not 100% sure of it, what do you think? I fear this section may be getting confusing.